modules/lib/src/main/resources/assets/js/app/ContentStatusToolbar.ts

```diff
@@ -56,9 +67,19 @@ export class ContentStatusToolbar
     private updateStatus(content: ContentSummaryAndCompareStatus): void {
         this.status.setClass('status');
 
+        if (this.content) {
+            const clazz = this.content.getStatusClass();
+            this.showChanges.removeClass(clazz);
+            this.status.removeClass(clazz);
+        }
+
+        this.content = content;
```

Comment:
We don't need `this.content` at all - `this.getItem()` will give us content previously stored by `this.setItem()` (see the suggestion above).